gui/src/components/Onboarding/Steps.tsx

```diff
@@ -101,6 +124,118 @@ function LiveBlockchainsStep({
   );
 }
 
+function CreateTestWalletStep({
+  formData,
+  setFormData,
+}: {
+  formData: WizardFormData;
+  setFormData: React.Dispatch<React.SetStateAction<WizardFormData>>;
+}) {
```

Comment:
please move these types to a `interface CreateTestWalletStepProps` above

It's personal preference, I know, but I find it really hard to parse function headers like this

Reply:
Addressed this [here](https://github.com/Subvisual-Academy/iron/pull/1/commits/776a779923d235c02f7e8798e758870995a61890).
This was happening at two more components which reenforces your point.
Looks much clearer and easier to read now, thanks for the suggestion 🙌